src/tractobots_robot_localization/params/ekf_localization.yaml

```diff
@@ -160,14 +168,11 @@ imu0_config: [false, false, false,
               false, false, true,
               false, false, false,
               false, false, true,
-              true,  false, false]
+              false, false, false]  # Only set accelerations to true if you have a velocity measurement
```

Comment:
Just what the comment says. Your accelerometers will have biases, which will get integrated by the filter, and cause the velocity to grow without bound. If you don't have something directly measuring velocity, you should leave these as false.